### PR TITLE
Results page

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my Results Page?

Live Link : https://water-quiz-rb41c1qx5-shlok30.vercel.app/

Video Demo:

https://user-images.githubusercontent.com/60795588/173008379-08607f08-d468-4dad-8c98-2f0aa0065d8a.mov

Thanks

### CODE REVIEW

**Review by srv-pranjal — APPROVED** [review]
UI looks great.
Results page is working as expected.
Remember to remove console.log statements in the end.
Good work.

File: src/pages/result.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                <h1 className="primary-colour-text">{currentQuizObject.title}</h1>
+                <p className="m3-top">Score : {calculateScore(answerList,quizID)}/15</p>
+                {currentQuizObject.mcqs.map((quiz,index) => {
+                    return(
+                        <>
+                            <div className="m3-top">
+                                <p className="text-l">{quiz.question}</p>
+                                <div className="flex flex-column gap-s m3-top">
+                                    {renderOptions(currentQuizObject,quiz.options,index)}
+                                </div>
```
Comment (nikhil-sharma-nks) [inline src/pages/result.js:28]: whenever you are rendering a component in a loop, it's good to have key attribute in it.

**Review by nikhil-sharma-nks — APPROVED** [review]
Looks good to merge. Just one suggestion please use a code formatter

File: src/pages/result.js (hunk 1 above)
Comment (shlok30) [reply src/pages/result.js:28]: Done, Thanks

**Pull request merged**